src/AppBundle/Document/TalkBlock.php

```diff
@@ -10,7 +10,9 @@
  */
 class TalkBlock extends SimpleBlock
 {
-    /** @PHPCR\Field(type="string") */
+    /**
+     * @PHPCR\String
```

Comment:
this is the old, deprecated format. we should use Field(type="string"). php 7 does not allow a class named String.

but we should make the docblock multiline like you did.